crates/wash-lib/src/start/wadm.rs

```diff
@@ -74,14 +74,25 @@ where
         // Check version to see if we need to download new one
         if let Ok(output) = Command::new(&wadm_bin_path).arg("--version").output().await {
             let stdout = String::from_utf8_lossy(&output.stdout).to_string();
-            if stdout.replace("wadm", "").trim() == version.trim_start_matches('v') {
+            print!("👀 Found wadm version: {}", stdout.trim_end());
+            let re = regex::Regex::new(r"^wadm[^\s]*").unwrap();     
+            if re.replace(&stdout, "").to_string().trim() == version.trim_start_matches('v') {
                 // wadm already exists, return early
+                println!(" Using it.✅");
                 return Ok(wadm_bin_path);
             }
```

Comment:
This should probably have a `print!` newline in an else block in case it doesn't find the right version

Reply:
Refactored!

Reply:
@thomastaylor312: I slightly adjusted the code.